SYSTEMS.md

```diff
@@ -193,3 +305,16 @@ Code:
 Result:
 
 ![Basic Example](doc_images/ex1.png)
+
+## **Notes for Forge Users:**
+  - best to use native Foundry until you are 100% certain you are happy with your template 
+  - uploads need to be a different file name every time
+  - 
+
+## **Notes for inexperienced template creators:**
+
+  - use an editor, not a simple text editor like notepad
+  - the problem is usually your json
+  - use the console
+  - finding the values for your systems data
```

Comment:
```suggestion
  - Double check that property values you're using are correct for your selected system.
```

Reply:
WIll also be adding ways I find those values for a selected system.